templates/contactList.html

```diff
@@ -13,4 +13,7 @@
 			<h2>{{ctrl.t.emptySearch}}</h2>
 		</div>
 	</div>
+	<div ng-if="filtered.length>0" class="contacts-list-count">
+		{{ filtered.length }} {{ filtered.length==1 ? ctrl.t.oneContact : ctrl.t.multipleContacts }}
```

Comment:
> where in the template is it used then?

It is used here:
https://github.com/owncloud/tasks/blob/master/templates/part.tasklist.php#L25

Using this as a filter may be more clear (and easier to test? :innocent:)